src/components/CartAside/CartAside.js

```diff
@@ -28,8 +37,13 @@ const CartAside = ({ cartItems }) => {
             <p className="cart-aside-total">
                {cartItems.length}
             </p>
-            {cartItems.map((item, index) =>
-               <CartAsideItem item={item} key={index} />)}
+            {cartItems.map((product, index) =>
+               <CartAsideItem
+                  deleteCartItemHandler={deleteCartItemHandler}
+                  product={product}
+                  key={index}
```

Comment:
Don't use the array index as a key.
https://medium.com/@robinpokorny/index-as-a-key-is-an-anti-pattern-e0349aece318

Reply:
Next sprint i will fix it :D